examples/kitchen-sink.yaml

```
@@ -87,6 +87,10 @@ logger_provider:
             #
             # Environment variable: OTEL_EXPORTER_OTLP_TIMEOUT, OTEL_EXPORTER_OTLP_LOGS_TIMEOUT
             timeout: 10000
+            # Configure client transport security for the exporter's connection.
+            #
+            # Environment variable: OTEL_EXPORTER_OTLP_INSECURE, OTEL_EXPORTER_OTLP_LOGS_INSECURE
+            insecure: true
```

```suggestion
            insecure: false
```
To reflect defaults from the spec?